src/main/kotlin/uk/gov/justice/hmpps/offendersearch/security/ResourceServerConfiguration.kt

```diff
@@ -19,7 +19,7 @@ class ResourceServerConfiguration : WebSecurityConfigurerAdapter() {
         .and().csrf().disable()
         .authorizeRequests { auth ->
           auth.antMatchers("/webjars/**", "/favicon.ico", "/csrf",
-              "/health", "/health/ping", "/info", "/health/liveness", "/health/readiness",
+              "/health", "/health/*", "/info",
```

Comment:
don't need `/health` and would be better to do a `/health/**` as it would then do nested too e.g. `/health/bob/joe`